### PR TITLE
#22907 Incorporate Bean Validation 3.0.0

### LINKED ISSUE #22907: Incorporate Bean Validation 3.0.0 into GlassFish 6.0

### PR DESCRIPTION
**Pull request by arjantijms**
Fixes #22907 It compiles locally, but that's about it.

### CODE REVIEW

**Comment by russgold** [comment]
Can you summarize what you've done here? Why do the tests now fail?

Also, please note that Oracle legal has changed their guidance on the copyright notices. A common is now needed after the last year, and "All rights reserved" should be removed.

**Comment by arjantijms** [comment]
> Can you summarize what you've done here? 

The summary is essentially that Bean Validation 3.0.0 has been integrated (incorporated). This means that the current version of bean validation in the pom has been updated to 3.0.0 and the implementation been updated to a version matching that. Since it changes the namespace from javax to jakarta, all the code that uses javax has been updated to jakarta.

> Why do the tests now fail?

They tests don't fail now, the tests have been failing for quite some time. See the comment here as well: https://github.com/eclipse-ee4j/glassfish/pull/22956

They are failing because there are a lot of dependencies that need to be updated, some circular.  If everyone creates draft PRs and tries to make them perfect and satisfy all other dependencies, it just builds-up until you have a couple of gigantic PRs that greatly overlap.

So therefor we have decided (see mailing-list) to allow failures for now, and then fix those once everything has been integrated.

**Review by smillidge — APPROVED** [review]
You've named the PRs the wrong way around. This is incorporate JSON-B the JSON-B one is Bean Validation

**Comment by smillidge** [comment]
> 
> 
> You've named the PRs the wrong way around. This is incorporate JSON-B the JSON-B one is Bean Validation

OK GitHub is messing with me. Now I see Bean Validation changes!

**Review by smillidge — APPROVED** [review]
